<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,5">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget">
        <item>
         <property name="text">
          <string>介绍</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验1：仿人足球机器人初探</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验2：Python编程实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验3：Ubuntu操作实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验4：ROS操作实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验5：Webots机器人动力学模型搭建实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验6：计算机视觉实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验7：手眼协同——位姿转换实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验8：机器人自主定位实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验9：机器人运动控制实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验10：机器人智能决策实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验11：仿人机器人仿真步行实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验12：搭建足球机器人仿真平台</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验13：仿人机器人步行速度控制实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验14：仿人机器人头部运动实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>实验15：仿人机器人足球识别实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>综合实验1：机器人点球实验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>综合实验2：多机器人自主对战实验</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5,2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_title">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>介绍</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <widget class="QWidget" name="page">
           <layout class="QHBoxLayout" name="horizontalLayout"/>
          </widget>
          <widget class="QWidget" name="page_1">
           <widget class="QVideoWidget" name="widget_1" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_2">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_3">
           <widget class="QVideoWidget" name="widget_3" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_4">
           <widget class="QWebEngineView" name="widget_4" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_5">
           <widget class="QWebEngineView" name="widget_5" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_6">
           <widget class="QWidget" name="widget_6" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_7">
           <widget class="QWidget" name="widget_7" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_8">
           <widget class="QWidget" name="widget_8" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_9">
           <widget class="QVideoWidget" name="widget_9" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_10">
           <widget class="QWidget" name="widget_10" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_11">
           <widget class="QWidget" name="widget_11" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_12">
           <widget class="QWidget" name="widget_12" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_13">
           <widget class="QWidget" name="widget_13" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_14">
           <widget class="QWidget" name="widget_14" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_15">
           <widget class="QWidget" name="widget_15" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_16">
           <widget class="QWidget" name="widget_16" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_17">
           <widget class="QWidget" name="widget_17" native="true">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>961</width>
              <height>541</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_descr">
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1520</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>仿人足球机器人课程</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtWebEngineWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
